MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -153,6 +154,46 @@ public override MvxBasePresentationAttribute CreatePresentationAttribute(Type vi
             return base.CreatePresentationAttribute(viewModelType, viewType);
         }
 
+        public override Task<bool> ChangePresentation(MvxPresentationHint hint)
```

Comment:
This method is long. Can we reduce complexity by making more smaller methods please?